nrf_security/configs/nrf-config.h.template

```diff
@@ -3394,7 +3394,7 @@ it is (2^48 - 1), our restriction is :  (int - 0xFFFF - 0xF).*/
 #cmakedefine MBEDTLS_ECP_FIXED_POINT_OPTIM @MBEDTLS_ECP_FIXED_POINT_OPTIM@ /**< Enable fixed-point speed-up */
 
 /* Entropy options */
-#define MBEDTLS_ENTROPY_MAX_SOURCES                  1   /**< Maximum number of sources supported */
+#cmakedefine MBEDTLS_ENTROPY_MAX_SOURCES             @MBEDTLS_ENTROPY_MAX_SOURCES@ /**< Maximum number of sources supported */
```

Comment:
Please add cmake-verification that this is not changed in case CryptoCell is enabled (non-secure of course always acceptable as it is SW only with SPM/PSA to call into secure domain)

Reply:
I have added the check and rebased the change

Reply:
If it is ok I will create PR to nrf to trigger test